cmd/cleanup-controller/utils/utils.go

```diff
@@ -10,7 +10,7 @@ import (
 
 func UnmarshalCleanupPolicy(kind string, raw []byte) (kyvernov1alpha1.CleanupPolicyInterface, error) {
 	var policy kyvernov1alpha1.CleanupPolicyInterface
-	if kind == "CleanupPolicy" {
+	if kind == "CleanupPolicy" || kind == "ClusterCleanupPolicy" {
```

Comment:
Both types share the same interface?

Reply:
It won't work, we can't unmarshal with an interface... we need tests @NikhilSharmaWe